src/main/java/domain/AllCrew.java

```diff
@@ -1,70 +1,87 @@
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.*;
+
+import static domain.ERROR_MESSAGE.NO_SUCH_NICKNAME;
 
 public class AllCrew {
     private final List<Crew> allCrew;
 
     public AllCrew() {
-        this.allCrew = new ArrayList<>();
+        allCrew = new ArrayList<Crew>();
     }
 
-    public void addCrew(Crew crew) {
-        allCrew.add(crew);
+    public AllCrew(Scanner fileScanner) {
+        allCrew = new ArrayList<>();
+        while (fileScanner.hasNextLine()) {
+            String line = fileScanner.nextLine();
+            List<String> crewNameAndAttendanceTime = List.of(line.split("-"));
+            String rawDateTime = crewNameAndAttendanceTime.get(1);
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy MM dd HH mm");
+            loadFileAllCrewData(rawDateTime, formatter, crewNameAndAttendanceTime);
+        }
     }
 
-    public boolean isContainedCrewName(String crewName) {
-        return allCrew.stream()
-                .anyMatch(crew -> crew.isSameName(crewName));
+    private void loadFileAllCrewData(String rawDateTime, DateTimeFormatter formatter, List<String> crewNameAndAttendanceTime) {
+        try {
+            LocalDateTime dateTime = LocalDateTime.parse(rawDateTime, formatter);
+            Attendance attendance = new Attendance(dateTime);
+            addCrewInfoWithNameAndAttendance(crewNameAndAttendanceTime.getFirst(), attendance);
+        } catch (DateTimeParseException e) {
+            throw new DateTimeException(ERROR_MESSAGE.INVALID_FILE_TIME_FORMAT.getMessage());
+        }
     }
 
-    public Attendance addCrewAttendanceByName(String name, LocalDateTime dateTime) {
-        Crew crew = findCrewByName(name);
-        return crew.addAttendance(dateTime);
+    public void addCrewInfoWithNameAndAttendance(String crewName, Attendance attendance) {
+        if (containsCrewName(crewName)) {
+            findCrewByName(crewName).addAttendance(attendance);
+            return;
+        }
+        Crew crew = new Crew(crewName);
+        crew.addAttendance(attendance);
+        allCrew.add(crew);
     }
 
-    public AttendanceUpdateResult modifyCrewAttendanceByName(String name, LocalDateTime dateTime) {
-        Crew crew = findCrewByName(name);
-        return crew.update(dateTime);
+    public boolean containsCrewName(String crewName) {
+        return allCrew.stream().anyMatch(crew -> crew.isSameName(crewName));
     }
 
-    public Crew findCrewByName(String name) {
-        return allCrew.stream()
-                .filter(crew -> crew.isSameName(name))
-                .findFirst()
-                .orElseThrow(() -> new IllegalArgumentException("존재하지 않는 크루입니다."));
+    public Crew findCrewByName(String crewName) {
+        for (Crew crew : allCrew) {
+            if (crew.isSameName(crewName))
+                return crew;
+        }
+        throw new NoSuchElementException(NO_SUCH_NICKNAME.getMessage());
```

Comment:
```suggestion
        return allCrew.stream()
                .filter(it -> it.isSameName(crewName))
                .findFirst()
                .orElseThrow(() -> new NoSuchElementException(NO_SUCH_NICKNAME.getMessage()));
```

stream을 사용하는 게 더 가독성이 좋냐, for문이 더 좋냐 하는 건 취향 차이가 크긴 하지만 이 정도 크기의 로직은 stream으로도 충분히 구현 가능해보여요. 특히나 인덴트가 2가 넘어가는 곳들은 더더욱이요! (이전엔 stream을 사용했었는데 for 문으로 변경된 이유가 있나요?)
물론 언제나 stream이 더 낫다는 것은 절대 아닙니다. 로직이 복잡하면 복잡할 수록 for문을 더 선호하는 편이긴 해요. 다른 곳들도 어떤 방식이 더 나을지 고민해보고 수정해볼까요?

추가적으로 줄바꿈도 컨벤션입니다. 대부분의 케이스에서는 한 줄에 점 하나만 찍는 게 가독성을 높여줘요. 그게 우테코에서 따르는 컨벤션이기도 하구요. 다른 곳들도 점검해주세요!

Reply:
지금까지는 stream과 for문에 크게 신경 쓰지 않았던 것 같고 익숙한 for문으로 작성 했던 것 같습니다.
stream을 사용하면 가독성이 확연히 개선되는 상황이거나 반환값이 존재하는 반복문의 경우 stream으로 변경을 의식하도록 하겠습니다!

줄바꿈 점검 및 수정하였습니다!